src/domain/event/eventInfo/otherEventTimes/OtherEventTimes.tsx

```diff
@@ -51,7 +52,15 @@ const OtherEventTimes: React.FC<Props> = ({
     history.push(eventUrl);
   };
 
-  if (!superEventId || events.length === 0) return null;
+  if (!superEventId || (events.length === 0 && !loading)) {
```

Comment:
Hmm to me this seems quite complex if statement for pretty simple thing, I think there is a way to make this much simpler and easier to understand

Reply:
hmm why is superEventId provided in the first place as a prop? I have no idea why I have done that

Reply:
I think that the superEventId prop can be omitted and then the code woul be something like this:

```
  if (loading) {
    return (
      <div className={styles.skeletonWrapper}>
        <SkeletonLoader />
      </div>
    );
  }

  if (events.length === 0) {
    return null;
  }

```

Reply:
i think its just logical: If superEventId is not provided -> do nothing, and then only some extra checks.

Reply:
i can fix it if its more readable though. and yes, in that case superEventId can be removed

Reply:
fixed